framework/PageActions/PageActionDropdown.tsx

```diff
@@ -165,9 +118,64 @@ export function PageActionDropdown<T extends object>(props: PageActionDropdownPr
     tooltipContent = undefined;
   }
 
+  const dropdownMenuLabel: string | JSX.Element | undefined =
+    iconOnly && CustomIcon ? (
+      <Icon>
+        <CustomIcon />
+      </Icon>
+    ) : (
+      label
+    );
+
   return (
     <Tooltip content={tooltipContent} trigger={tooltipContent ? undefined : 'manual'}>
-      {dropdown}
+      <Dropdown
+        isOpen={dropdownOpen}
+        onSelect={() => setDropdownOpen(false)}
+        onOpenChange={(isOpen) => setDropdownOpen(isOpen)}
+        popperProps={{
```

Comment:
When the dropdown opens near the bottom of a table, it causes the whole page to get a scroll bar.
The fix is to mount the dropdown at the document.body.
We can also leverage the preventOverflow in that case to auto position the dropdown.
Then we can remove the position prop from the component.
```
        popperProps={{
          appendTo: () => document.body,
          preventOverflow: true,
          enableFlip: true,
        }}
```
This may affect tests, but fixes the problem with the overflow scrolling!